kiwixbuild/builder.py

```diff
@@ -144,6 +152,19 @@ def _get_packages(self):
                 if packages is not True:
                     # True means "assume the dependency is install but do not try to install anything for it"
                     packages_list += packages
+
+            # get {host}_{codename}_{config} packages
+            mapper_name = "{host}_{codename}_{config}".format(
+                host=neutralEnv("distname"), codename=neutralEnv("codename"), config=configName
+            )
+            package_name_mapper = PACKAGE_NAME_MAPPERS.get(mapper_name, {})
+            packages = package_name_mapper.get(builderName)
+            if packages:
+                to_drop.append(builderDef)
+                if packages is not True:
+                    # True means "assume the dependency is install but do not try to install anything for it"
+                    packages_list += packages
+
```

Comment:
Get rid of this code duplication taking advantage of `_get_mapper_names_for_config()` like proposed in the previous comment.